src/osgEarth/Utils

```diff
@@ -252,6 +259,129 @@ namespace osgEarth
         virtual ~AllocateAndMergeBufferObjectsVisitor() { }
         void apply(osg::Geode& geode);
     };
+
+    inline osg::ref_ptr<osg::Object> readObjectFile(const std::string& filename, const osgDB::Options* options)
+    {
+#if OSG_VERSION_GREATER_OR_EQUAL(3,4,0)
+        return osgDB::readRefObjectFile(filename, options);
+#else
+        return osgDB::readObjectFile(filename, options);
+#endif
+    }
+
+    inline osg::ref_ptr<osg::Object> readObjectFile(const std::string& filename)
+    {
+#if OSG_VERSION_GREATER_OR_EQUAL(3,4,0)
+        return osgDB::readRefObjectFile(filename);
+#else
+        return osgDB::readObjectFile(filename);
+#endif
+    }
+
+    template<typename T>
+    inline osg::ref_ptr<T> readFile(const std::string& filename, const osgDB::Options* options)
+    {
+#if OSG_VERSION_GREATER_OR_EQUAL(3,5,1)
+        return osgDB::readRefFile<T>(filename, options);
```

Comment:
readRefFile (a templatized version of readRefObjectFile) was introduced in osg 3.5.1.